openmp/runtime/src/include/ompx.h.var

```diff
@@ -9,6 +9,8 @@
 #ifndef __OMPX_H
 #define __OMPX_H
 
+typedef unsigned long uint64_t;
+
```

Comment:
I wish we could just include `stdint.h` here, but we'd need to make sure it only uses the one in the `clang` resource directory. That's doable if we don't set `__STDC_HOSTED__` or use `-ffreestanding`. Using `-ffreestanding` would be an option if I could fix the bug that prevents me from landing the fixes that allow inlining.